README.md

```diff
@@ -93,7 +93,11 @@ https://github.com/Unity-Technologies/ml-agents.
 
 ## Additional Resources
 
-We have published a series of blog posts that are relevant for ML-Agents:
+We have a Unity Learn course,
+[ML-Agents: Hummingsbird](https://learn.unity.com/course/ml-agents-hummingbirds),
+that provides a gentle introduction to Unity and the ML-Agents Toolkit.
```

Comment:
gentle?

Reply:
Intended to highlight that it's an easy to follow tutorial.